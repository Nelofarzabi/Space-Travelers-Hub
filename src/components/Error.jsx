import React from 'react';
import './Loading.css';

const Error = () => (
  <div className="loading--container">
    <h4 className="Loading--header">An Error ocuured</h4>
  </div>
);

export default Error;
