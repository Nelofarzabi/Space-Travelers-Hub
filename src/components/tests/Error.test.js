import { render } from '@testing-library/react';
import Error from '../Error';

describe('test error component', () => {
  test('Error renders correctly', () => {
    const { container } = render(<Error />);
    expect(container).toMatchSnapshot();
  });
});
