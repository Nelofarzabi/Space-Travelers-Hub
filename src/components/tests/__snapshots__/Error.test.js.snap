// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test error component Error renders correctly 1`] = `
<div>
  <div
    class="loading--container"
  >
    <h4
      class="Loading--header"
    >
      An Error ocuured
    </h4>
  </div>
</div>
`;
